Dockerfile

```diff
@@ -37,7 +37,7 @@ RUN pip install --trusted-host pypi.org --trusted-host files.pythonhosted.org -r
 # Run app with reload option
 EXPOSE 8080
 CMD if [ "${PATH_TO_OTEAPI_CORE}" != "/dev/null" ] && [ -n "${PATH_TO_OTEAPI_CORE}" ]; then \
-  pip install -U --force-reinstall -c requirements.txt -e /oteapi_core; fi \
+  pip install -U --force-reinstall -e /oteapi_core; fi \
```

Comment:
Why is requirements removed?

Reply:
It's removed as a constraint for the installation of `oteapi-core`. This is to avoid the installation failing, should the local version of `oteapi-core` differ from the one listed in `requirements.txt`.
Essentially, keeping the `-c` command here defeats the purpose of installing a local version of `oteapi-core`. I.e., this was really a bug.